provision/prepare_template

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
-cp template.yaml instances.yaml
-sed  -i "s|@localip|$1|g" "instances.yaml"
-sed  -i "s|@routerip|$2|g" "instances.yaml"
+cp ../../$3/$4/$5/template.yaml ../../$3/$4/$5/instances.yaml
```

Comment:
Please provide this path in one argument.

Reply:
Done.